.github/workflows/deploy.yml

```diff
@@ -0,0 +1,86 @@
+name: Deploy
+
+on:
+  push:
+    tags: 
+      - '*'
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        
+      - name: Cache node modules
+        uses: actions/cache@v1
+        id: cache
+        with:
+          path: node_modules
+          key: node-modules-${{ hashFiles('yarn.lock') }}
+
+      - name: Instal dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
+
+      - name: Cache node modules @ example
+        uses: actions/cache@v1
+        id: cache-example
+        with:
+          path: ./example/node_modules
+          key: node-modules-example--${{ hashFiles('example/yarn.lock') }}
+
+      - name: Instal dependencies @ example
+        if: steps.cache-example.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile --cwd ./example
+
+      - name: Cache node modules @ website
+        uses: actions/cache@v1
+        id: cache-website
+        with:
+          path: ./website/node_modules
+          key: node-modules-website-${{ hashFiles('website/yarn.lock') }}
+
+      - name: Instal dependencies @ website
+        if: steps.cache-website.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile --cwd ./website
+      
+  webiste:
+    name: Deploy website
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout  
+        uses: actions/checkout@v2
+ 
+      - name: Cache node modules
+        uses: actions/cache@v1
+        id: cache
+        with:
+          path: node_modules
+          key: node-modules-${{ hashFiles('yarn.lock') }
+
+      - name: Cache node modules @ website
+        uses: actions/cache@v1
+        id: cache-website
+        with:
+          path: ./website/node_modules
+          key: node-modules-website-${{ hashFiles('website/yarn.lock') }}
+
+      - name: Cache node modules @ example
+        uses: actions/cache@v1
+        id: cache-example
+        with:
+          path: ./example/node_modules
+          key: node-modules-example--${{ hashFiles('example/yarn.lock') }}
+          
+      - name: Deploy to GH Pages
+        run: |
+            git config --global user.name "adamtrz@users.noreply.github.com"
+            git config --global user.email "trzcinski.adam@gmail.com"
```

Comment:
what about storing this info in envs?
https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets

Reply:
yeah, already using secrets so why not 😉